CONTRIBUTING.md

```diff
@@ -14,6 +14,7 @@ We welcome you to use the GitHub issue tracker to report bugs or suggest feature
 When filing an issue, please check [existing open](https://github.com/aws/aws-encryption-sdk-c/issues), or [recently closed](https://github.com/aws/aws-encryption-sdk-c/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), issues to make sure somebody else hasn't already 
 reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
 
+* If you discover a security critical bug, please report it via http://aws.amazon.com/security/vulnerability-reporting/ and **do not** create a public issue. 
```

Comment:
Thank you for adding this! I would please pull straight from https://github.com/amzn/.github/blob/master/CONTRIBUTING.md#security-issue-notifications and make it a heading so that it really pops out. It's worth the space to make it easy for people to discover, IMHO.

Reply:
updated.